### ISSUE TITLE
java/beans/PropertyEditor/6380849/TestPropertyEditor.java test fail on linux

### ISSUE THREAD

**Issue by sendaoYan** [adoptium/aqa-tests#3671]
**Describe the bug**
The testcase java/beans/PropertyEditor/6380849/TestPropertyEditor.java from extended.openjdk jdk_beans_0 target fail because of "expected editor is not found"

**To Reproduce**
Steps to reproduce the behavior (or Grinder rerun link):
jtreg -v:fail,error -w tmp -nr openjdk/openjdk-jdk/test/jdk/java/beans/PropertyEditor/6380849/TestPropertyEditor.java

**Expected behavior**
test pass, or excluded by Problemlist.txt

**Screenshots**

this testcase only exclude for macos now.

![image](https://user-images.githubusercontent.com/24123821/169034935-d7d8b9b1-ba06-402e-a77c-773d66a34876.png)

**Additional context**

**Comment by sendaoYan** [comment]
similar testcase:

```
java/beans/PropertyEditor/6380849/TestPropertyEditor.java
java/beans/PropertyEditor/TestColorClassNull.java
java/beans/PropertyEditor/TestColorClassJava.java
java/beans/PropertyEditor/TestColorClassValue.java
java/beans/PropertyEditor/TestColorClass.java
java/beans/PropertyEditor/TestFontClassJava.java
java/beans/PropertyEditor/TestFontClassNull.java
java/beans/PropertyEditor/TestFontClass.java
java/beans/PropertyEditor/TestFontClassValue.java
```

**Comment by sendaoYan** [comment]
After set DISPLY, these tests run pass. Do we need update the document for "Run AQAvit via Command-line" or "[test prerequisites](https://github.com/adoptium/aqa-tests/blob/master/doc/Prerequisites.md)"?

**Comment by smlambert** [comment]
Yes we can update the Run AQAvit docs, DISPLAY is needed to be set.
